docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,160 @@
+---
+keywords: [intermediate, governance, concept, nns]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Neuron management
+
+<MarkdownChipRow labels={["Intermediate", "Governance", "Concept"]} />
+
+## Overview 
+This page describes the technical APIs and details to manage neurons on the NNS governance canister.
```

Comment:
```suggestion
This page describes the technical APIs used to manage neurons on the NNS.
```

Reply:
Applied in local copy and will be commited in next version.